config/vendor-api-v1.yml

```diff
@@ -464,6 +465,11 @@ components:
           description: The candidate’s ID in the Apply system
           maxLength: 10
           example: C5432
+        application_reference:
+          type: string
+          description: The candidate’s reference in the Apply system
```

Comment:
This suggests the reference is a pointer to the candidate rather than a reference for them to use. Consider something more explicit, like "The candidate's reference number for their application in the Apply system"?